### ISSUE TITLE
Support importing modules from npm package

### ISSUE THREAD

**Issue by jeremydw** [stevenlei/spacingjs#22]
We (https://github.com/blinkk) would like to include SpacingJS in our app and need to be able to import it with the following syntax:

```
import Spacing from './spacing/spacing';
```

Most projects do not distribute the `.ts` files, but rather the generated `.js` files instead. Looking at this project, only the `.ts` files are included. We'd like the npm package to include the generated `.js` files so we can import them directly within our project:

```
$ npm pack
npm notice 
npm notice 📦  spacingjs@1.0.7
npm notice === Tarball Contents === 
npm notice 67B     .babelrc                                  
npm notice 1.1kB   LICENSE                                   
npm notice 1.9kB   README.md                                 
npm notice 1.9kB   README.zh-Hans.md                         
npm notice 1.9kB   README.zh-Hant.md                         
npm notice 7.2kB   dist/bundle.js                            
npm notice 1.2kB   extensions.config.js                      
npm notice 148B    extensions/chrome/background.js           
npm notice 7.2kB   extensions/chrome/bundle.js               
npm notice 627B    extensions/chrome/icon16.png              
npm notice 1.4kB   extensions/chrome/icon48.png              
npm notice 4.2kB   extensions/chrome/icon128.png             
npm notice 500B    extensions/chrome/manifest.json           
npm notice 362B    extensions/tampermonkey/header.js         
npm notice 7.6kB   extensions/tampermonkey/spacingjs_1.0.7.js
npm notice 4.7kB   index.html                                
npm notice 1.0kB   package.json                              
npm notice 115B    prettier.config.js                        
npm notice 252.1kB screenshot.png                            
npm notice 214.6kB spacingjs-1.0.7.tgz                       
npm notice 69B     src/index.ts                              
npm notice 5.6kB   src/marker.ts                             
npm notice 1.9kB   src/placeholder.ts                        
npm notice 1.0kB   src/rect.ts                               
npm notice 5.6kB   src/spacing.ts                            
npm notice 208B    src/type.ts                               
npm notice 581B    tsconfig.json                             
npm notice 1.1kB   webpack.config.js                 
```
Basically, we're expecting `src/index.js` or `dist/index.js` files to exist above.

**Comment by stevenlei** [comment]
Hello, sorry for my late reply and thanks for your comments, I will follow up.
